exe_1.py

```diff
@@ -0,0 +1,29 @@
+import sys
+
+def findPermutations(str1, str2):
+	str1 = str1.lower()
+	str2 = str2.lower()
+
+	letters_and_frequencies_1 = make_letters_and_frequencies(str1)
+	letters_and_frequencies_2 = make_letters_and_frequencies(str2)
+	return compare_dicts(letters_and_frequencies_1, letters_and_frequencies_2)
+
+def make_letters_and_frequencies(string):
+	lf = {}
+	for char in string:
+		if char in lf:
+			lf[char] += 1
+		else:
+			lf[char] = 1
+	return lf
+
+def compare_dicts(dict1, dict2):
```

Comment:
I could keep two counters of the length of each string, and first pre-check that, since if the length is the same and then some char doesn't exist in the dict2 (if we use only first comparison) ,  then it should return False.